be/src/exec/pipeline/scan/connector_scan_operator.cpp

```diff
@@ -480,47 +477,25 @@ int ConnectorScanOperator::available_pickup_morsel_count() {
 
     // adjust io tasks according information collected
     P.cs_gen_chunks_time = (now - P.op_start_time - P.cs_total_halt_time);
+
     // chunk sources in each scan operator don't generate chunks in uniform speed
     // but chunks are distributed evenly to scan operator. So we I think we'd better not to
     // use chunks generated by chunks source issues by this scan operator, but to use
     // chunks generated by all scan operators. A approximate value is to use `op_pull_chunks`
     // produced chunks per 10ms, but cs running time is microsecond unit.
-    double balanced_cs_speed = _op_pull_chunks * 10000.0 / (P.cs_gen_chunks_time + 1);
+    double source_speed = _op_pull_chunks * 1e4 / (P.cs_gen_chunks_time + 1);
```

Comment:
a lot of literal numbers here, not sure if there is a better way to make the code more human readable.

Reply:
No, because we have to align time unit here, like some variables are ns and some are ms,us. we have to align them.